src/osd/OSD.cc

```diff
@@ -160,6 +161,8 @@
 #undef dout_prefix
 #define dout_prefix _prefix(_dout, whoami, get_osdmap_epoch())
 
+using namespace std::chrono;
```

Comment:
this exposes lots of names like `seconds`,`hours`. i'd suggest just `using` the used types or keep using the `chrono::` namespace.